modules/mobile/app/models/mobile/v0/suggested_address.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'common/models/resource'
+
+module Mobile
+  module V0
+    class SuggestedAddress < Common::Resource
+      attribute :id, Types::String
+      attribute :address_line1, Types::String.optional.default(nil)
```

Comment:
I don't love how defensive this is, but I couldn't be sure that anything other than id would be present.